provision/acc_provision/templates/aci-containers.yaml

```diff
@@ -1543,7 +1543,7 @@ data:
           {% endif %}
         },
         "prometheus": {
-            "enabled": "false"
+            "enabled": {{ config.kube_config.opflex_agent_prometheus|json }}
```

Comment:
I believe this is being read as a string on the aci-containers side, right now its going as a boolean and its probably not going to work. Essentially your generated yamls in the UTs should not have a diff on this line where its false, and if they do, something is not right.

Reply:
@snaiksat did modify the PR according to str vs boolean..
Just wanted to bring this up for this PR, we only need corresponding PR in support repo for fix-testdata which is already present and nothing is required to change on aci-containers side as per my understanding.

Reply:
Yes, no change will be required on aci-containers side if we do the change here correctly (meaning that the value's type has to match what is currently expected in aci-containers).